package.json

```diff
@@ -61,7 +61,7 @@
     "@typescript-eslint/parser": "^4.29.0",
     "broccoli-babel-preset-typescript": "^1.0.1",
     "broccoli-merge-trees": "^4.2.0",
-    "documentation": "^13.2.5",
+    "documentation": "npm:@buschtoens/documentation@13.2.6",
```

Comment:
We may want to consider taking this opportunity to switch to a more well-maintained tool, like [API Extractor](https://api-extractor.com/). That would also improve the `.d.ts` build process.

Reply:
@chriskrycho

- Should I keep pointing at my fork here?
- Or should we switch to a different tool like [API Extractor](https://api-extractor.com/) first? If so, ...
  - Which tool would prefer?
  - Should this be split out into an extra PR that is merged before this one?
- Or do you have a completely different solution to this in mind?

Reply:
I think you're exactly right directionally! Let's see if we can get [TypeDoc](https://typedoc.org) wired up to do what we want. TypeDoc is almost certainly what we're going to end up using for Ember itself; it is how we are doing *all* of our TS-powered docs-generation at LinkedIn; and it is what I've used most successfully on other TS projects.

(API Extractor is (a) barely maintained at this point and (b) uninterested in supporting anything other than single-entry-point Node-style packages. TypeDoc is well-maintained, open to contributions, and actively working on supporting things like package `exports` maps. If we find gaps, let's help fix them there!)